frontend/public/components/resource-quota.jsx

```diff
@@ -191,16 +191,24 @@ const NoQuotaGuage = ({ title, className }) => (
 
 export const QuotaGaugeCharts = ({ quota, resourceTypes, chartClassName = null }) => {
   const resourceTypesSet = new Set(resourceTypes);
+  const cpuRequestUsagePercent = getResourceUsage(
+    quota,
+    resourceTypesSet.has('requests.cpu') ? 'requests.cpu' : 'cpu',
+  ).percent;
+  const cpuLimitUsagePercent = getResourceUsage(quota, 'limits.cpu').percent;
+  const memoryRequestUsagePercent = getResourceUsage(
+    quota,
+    resourceTypesSet.has('requests.memory') ? 'requests.memory' : 'memory',
+  ).percent;
+  const memoryLimitUsagePercent = getResourceUsage(quota, 'limits.memory').percent;
   return (
     <div className="co-resource-quota-chart-row">
       {resourceTypesSet.has('requests.cpu') || resourceTypesSet.has('cpu') ? (
         <div className="co-resource-quota-gauge-chart">
           <GaugeChart
             data={{
-              y: getResourceUsage(
-                quota,
-                resourceTypesSet.has('requests.cpu') ? 'requests.cpu' : 'cpu',
-              ).percent,
+              x: `${cpuRequestUsagePercent}%`,
+              y: cpuRequestUsagePercent,
```

Comment:
Is there a reason we append `%` for `x` and not for `y`?

Reply:
`y' is the actual numeric data, `x` is the category string/label, which is used in the tooltip.  
https://formidable.com/open-source/victory/docs/victory-pie/